Resources/public/js/ajax-updateOnChange.js

```diff
@@ -0,0 +1,36 @@
+$(document).on(
```

Comment:
Please add some documentation here.
Fill at least the @description and the purpose of this lib